elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -0,0 +1,354 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  // Padding: calc(#{pfe-var(container-spacer)} * 2),
+  // BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  // Disable accordion's active borders
+  --pfe-accordion--BackgroundColor: transparent;
+  --pfe-accordion--BorderColor: #ccc;
+  --pfe-accordion--BorderColor--accent: transparent;
+  --pfe-accordion--accent: transparent;
+  --pfe-theme--color--surface--border: transparent;
+  --pfe-theme--color--ui-base--on-dark: transparent;
+  --pfe-theme--color--ui-base: transparent;
+  button{
+    text-decoration: none;
+  }
+  font-weight: #{pfe-var(font-weight--normal)};
+  display: block;
+  position: sticky;
+  top: calc(#{pfe-var(container-spacer)} * 4);
+  @media(min-width: 992px) {
+    padding: calc(#{pfe-var(container-spacer)} * 2);
+  }
+}
+
+::slotted([slot="heading"]) {
+  margin: 0;
+  margin-bottom: calc(#{pfe-var(container-spacer)} * 1);
+  font-size: #{pfe-var(font-size--heading--zeta)};
+  font-weight: #{pfe-var(font-weight--normal)};
+  text-transform: uppercase;
+  @media(max-width: 992px) {
+    display: none;
+  }
+}
+
+ul{
+  &.pfe-jump-links-nav {
+    &[horizontal] {
+
+    }
+    padding: calc(#{pfe-var(container-spacer)} * 1);
+    padding-left: 0;
+    margin: 0;
+    list-style: none;
+  }
+  &.sub-nav {
+    margin: 0;
+    padding: 0;
+    overflow-y: hidden;
+    transition: flex 1s linear;
+    display: flex;
+    flex-direction: column;
+    flex: 0;
+  }
+}
+
+li {
```

Comment:
Can you nest the li styles a bit to help clarify the relationship between the classes?